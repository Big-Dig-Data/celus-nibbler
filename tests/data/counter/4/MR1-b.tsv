Collection	Content Provider	Platform	Reporting Period Total	Jan-2018	Feb-2018	Mar-2018	Apr-2018	May-2018	Jun-2018	Jul-2018	Aug-2018	Sep-2018	Oct-2018	Nov-2018	Dec-2018
Total for all collections		MyPlatform	15	0	3	0	1	10	0	0	0	0	0	0	1
ItemName1	MyPublisher	MyPlatform	11	0	0	0	1	10	0	0	0	0	0	0	0
ItemName2	MyPublisher	MyPlatform	4	0	3	0	0	0	0	0	0	0	0	0	1
