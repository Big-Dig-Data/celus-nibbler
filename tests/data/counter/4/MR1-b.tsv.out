MR1-a.tsv.out